api.md

```diff
@@ -333,6 +334,20 @@ if `conditions` is already a hypertable:
 SELECT create_hypertable('conditions', 'time', if_not_exists => TRUE);
 ```
 
+Convert table `measurements`, containing a composite type `report` which contains a field named `created` to hypertable with just time partitioning.
```

Comment:
Maybe don't use `created` as an example field?  It might confuse the issue with the actual `create_hypertable` command.

Reply:
I would say something like:

```
Time partition table `meausurements` on a composite column type `report` using a time partitioning function:
```